package.json

```diff
@@ -254,7 +254,8 @@
             "text-summary",
             "lcov"
         ],
-        "testResultsProcessor": "@casualbot/jest-sonar-reporter"
+        "testResultsProcessor": "@casualbot/jest-sonar-reporter",
+        "cacheDirectory": "/tmp/jest_cache"
```

Comment:
This doesn't exist on Windows so will likely cause issues

Reply:
It's an attempt to see if the CI is running faster with jest cache. If there is improvement in execution time, I'll see how make it works an all platforms

Reply:
Can probably be passed as a CLI option specifically in the CI workflow ftr

Reply:
Moved it to the CI workflow